.github/dependabot.yml

```diff
@@ -37,3 +37,13 @@ updates:
       - dependency-name: "k8s.io/mount-utils"
       - dependency-name: "k8s.io/pod-security-admission"
       - dependency-name: "k8s.io/sample-apiserver"
+  - package-ecosystem: "gomod"
+    directory: "/actions/retest"
+    schedule:
+      interval: "weekly"
+    rebase-strategy: "disabled"
+    labels:
+      - rebase
+      - ci/skip/e2e
```

Comment:
@nixpanic add this label in another place(top section) also in this file?

Reply:
for the main code in the repository we do want to run the full e2e tests, those depend on the vendored packages too, and we need to prevent breakage

Reply:
ah, i miss reading the label. I was thinking of it as `/retest/e2e`